src/main/java/org/folio/circulation/support/Clients.java

```diff
@@ -338,6 +340,10 @@ public CollectionResourceClient noticeTemplatesClient() {
     return noticeTemplatesClient;
   }
 
+  public CollectionResourceClient actualCostRecordsStorage() {
+    return actualCostRecordsStorageClient;
+  }
+
```

Comment:
Why is this part added here? Should we move it up where the storages are created?

Reply:
It is moved up, thanks.